torch_xla/distributed/xla_dist.py

```diff
@@ -362,9 +362,11 @@ def _prepare_scripts(self, cmd):
       if self.tpuvm_mode:
         # Start the local tf server if it is not already running.
         script.append([
-            'python', '-m', self.XRT_RUN_SERVER_CMD,
+            'python', '-m', self.XRT_RUN_SERVER_CMD, '--port',
```

Comment:
Now the `XRT_RUN_SERVER_PROCESS` constant in this file might be unused

Reply:
Ah, we actually use check in https://github.com/pytorch/xla/blob/master/third_party/xla_client/xrt_computation_client.cc#L165 to not start the local server for the pod.